src/pymodaq_plugins_newport/daq_move_plugins/daq_move_Newport_XPS_Q8.py

```diff
@@ -0,0 +1,294 @@
+from pymodaq.control_modules.move_utility_classes import DAQ_Move_base, comon_parameters_fun, main, DataActuatorType,\
+    DataActuator  # common set of parameters for all actuators
+from pymodaq.utils.daq_utils import ThreadCommand # object used to send info back to the main thread
+from pymodaq.utils.parameter import Parameter
+from qtpy.QtCore import QThread
+from pymodaq_plugins_newport.hardware import XPS_Q8_drivers 
+import sys
+
+from time import perf_counter_ns
+
+class XPSPythonWrapper():
+    """Simplified XPS wrapper, calls methods from the wrapper given by Newport. See XPS_Q8_drivers"""
+    
+    def __init__(self, ip:str = None, port:int = None, group:str = None, positionner:str = None, plugin = None):
+        #init the wrapper given by Newport and some attributes
+        self.myxps = XPS_Q8_drivers.XPS()   #Instanciate the driver from Newport
+        
+        #keep a ref of the plugin to emit (error) messages
+        self._plugin = plugin 
+        
+        #required to connect via TCP/IP
+        self._ip = ip
+        self._port = port
+        self.socketId = -1
+        
+        #Definition of the stage
+        self._group = group 
+        self._positioner = positionner
+        self._full_positionner_name = f'{group}.{positionner}'
+        
+        #Some required initialisation steps
+        self._initCommands()
+            
+    def _initCommands(self):
+        """Runs some initial commands : connect to the XPS server, group kill, group intialize, move home.
+        Some configs could be added here as well"""
+        self.socketId = self.myxps.TCP_ConnectToServer(self._ip, self._port, 20)    #20s timeout
+        # Check connection passed
+        if (self.socketId == -1):
+            self._plugin.emit_status(ThreadCommand('Update_Status', ['Connection to XPS failed, check IP & Port']))
+        else:
+            self._plugin.emit_status(ThreadCommand('Update_Status', ['Connected to XPS']))
+        
+            #Group kill to be sure
+            [errorCode, returnString] = self.myxps.GroupKill(self.socketId, self._group)
+            if (errorCode != 0):
+                self.displayErrorAndClose(errorCode, 'GroupKill')
+
+            #Initialize
+            [errorCode, returnString] = self.myxps.GroupInitialize(self.socketId, self._group) 
+            if (errorCode != 0):
+                self.displayErrorAndClose(errorCode, 'GroupInitialize')
+            
+            #Home search
+            self.moveHome()
```

Comment:
use only lowercase, please apply this everywhere

Reply:
done mainly on commits [e909d03](https://github.com/PyMoDAQ/pymodaq_plugins_newport/pull/6/commits/e909d033ddb3efb4524667b5bc990bc3cf0de203) & [f0e37fd](https://github.com/PyMoDAQ/pymodaq_plugins_newport/pull/6/commits/f0e37fde78a425566ce42e16cc4371c136766337)